### ISSUE TITLE
Investigate acceptance test fails in Uppy PR 6202

### ISSUE THREAD

**Issue by phil-davis** [owncloud/web#6807]
PR #6202 implements changes to the code that web uses to do file uploads.

Example: https://drone.owncloud.com/owncloud/web/24835/28/20

Some acceptance tests are failing, mostly related to steps like:
```
And as "Alice" the content of "/Shares/new-lorem.txt" in the server should be the same as the content of local file "new-lorem.txt"
    ✖ failed
      Error: AssertionError [ERR_ASSERTION]: asserting content of local file "/uploads/new-lorem.txt"
```

that come after a step like:
```
And the user uploads overwriting file "new-lorem.txt" using the webUI
```

Maybe "the user uploads overwriting file" is not working correctly (there might be different ids/paths for the needed UI elements), or the middleware is no longer correctly finding the file in the server, or not finding the local file to compare it with???

Investigate and fix.

**Comment by phil-davis** [comment]
The e2e tests have some diffs that made them pass:
https://github.com/owncloud/web/pull/6202/files#diff-2ee226f1addf8684ac43837c0e07687db26b006ea5b1236508b177f4d5423177

https://github.com/owncloud/web/pull/6202/files#diff-04799a6dcff8f98c11e8c02157db06456acafafc398a23200167b86e08e094b3

Those might give a clue about changes needed to UI element selectors etc.

**Comment by pascalwengerter** [comment]
Closed via #6202
